### PR TITLE
Stage - Implement multi-language in giddh v3

### PR DESCRIPTION
**Pull request by ravinderwalkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html [hunk 1]
```diff
@@ -1,908 +1,938 @@
...
-                                    </div>
-                                    <div class="form-group col-xl-2 col-md-6 payment-col">
-                                        <label>Account Number <span class="required">*</span></label>
-                                        <br>
-                                        <input type="text" placeholder="Enter Account Number" class="form-control" disabled formControlName="accountNo" name="accountNo" autocomplete="off" />
-                                    </div>
-                                    <div class="form-group col-xl-2 col-md-6 payment-col">
-                                        <label>Alias ID</label>
-                                        <br>
-                                        <input type="text" placeholder="Enter Alias ID" class="form-control" name="aliasId" disabled formControlName="aliasId" />
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.html:420]: Need to confirm about this.

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html [hunk 2]
```diff
@@ -1,908 +1,938 @@
...
-                                    </div>
-                                    <div class="form-group col-xl-2 col-md-6 payment-col">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.html:422]: Need to confirm about this.

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html [hunk 3]
```diff
@@ -1,908 +1,938 @@
...
-                                        <label>Accounts <span class="required">*</span></label>
-                                        <br>
-                                        <sh-select #paymentFormAccountName [options]="bankAccounts$ | async" name="accountUniqueName" formControlName="accountUniqueName" placeholder="Select Account" [ItemHeight]="33" autocomplete="off" [isRequired]="true">
-                                        </sh-select>
-                                    </div>
-                                </div>
-                                <ng-container formArrayName="userAmountRanges">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.html:429]: Need to confirm about this.

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html [hunk 4]
```diff
@@ -1,908 +1,938 @@
...
                                 <div class="form-group col-sm-4 col-xl-2 col-md-6 payment-col">
                                     <label>Account Number <span class="required">*</span></label>
                                     <br>
-
-                                    <input type="text" placeholder="Enter Account Number" name="accountNo" required class="form-control" [(ngModel)]="regAcc.accountNo" disabled />
-
+                                    <input type="text" placeholder="Enter Account Number" class="form-control" [(ngModel)]="paymentFormObj.accountNo" name="accountNo" required autocomplete="off" />
                                 </div>
                                 <div class="form-group col-sm-4 col-xl-2 col-md-6 payment-col">
                                     <label>Alias ID</label>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.html:545]: Need to confirm about this.

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html [hunk 5]
```diff
@@ -1,908 +1,938 @@
...
                                     <br>
-
-                                    <input type="text" placeholder="Enter Alias ID" name="aliasId" class="form-control" [(ngModel)]="regAcc.aliasId" disabled />
-
+                                    <input type="text" placeholder="Enter Alias ID" class="form-control" name="aliasId" [(ngModel)]="paymentFormObj.aliasId" />
                                 </div>
-                                <div class="form-group col-sm-4 col-xl-2 col-md-6 payment-col" *ngIf="bankAccounts$ && regAcc?.account?.uniqueName">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.html:551]: Need to confirm about this.

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html [hunk 6]
```diff
@@ -1,908 +1,938 @@
...
+                                                    <input type="checkbox" (click)="selectUsers(user, $event, userIndex)"
+                                                        [checked]="user.isSelected" /> {{user.label}}
+                                                </li>
+                                            </ul>
+                                        </div>
+                                        <!-- <sh-select [disabled]="isBankUpdateInEdit !== parentIndex" [options]="usersList" name="usersList" [defaultValue]="regAcc?.combinedUserNames" [(ngModel)]="regAcc.emailId" placeholder="Select User" [ItemHeight]="33" autocomplete="off"   (selected)="selectUser($event)"></sh-select> -->
+                                    </div>
+                                </div>
+                                <!-- Added this new row make it dynamic -->
+                                <!-- <ng-container *ngFor="let item of regAcc.userAmountRanges; let index = index; let firstIndex = first; let lastIndex = last">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.html:577]: Need to confirm about this.

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html [hunk 7]
```diff
@@ -1,908 +1,938 @@
...
-                                <br>
-                                <input type="text" placeholder="Enter User ID" class="form-control" formControlName="userId" name="userId" required autocomplete="off" />
-                            </div>
-                            <div class="form-group col-sm-4 col-xl-2 col-md-6 payment-col">
-                                <label>Account Number <span class="required">*</span></label>
-                                <br>
-                                <input type="text" placeholder="Enter Account Number" class="form-control" formControlName="accountNo" name="accountNo" autocomplete="off" />
-                            </div>
-                            <div class="form-group col-sm-6 col-xl-2 col-md-6 payment-col">
-                                <label>Alias ID</label>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.html:791]: Need to confirm

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html [hunk 8]
```diff
@@ -1,908 +1,938 @@
...
-                                <br>
-                                <input type="text" placeholder="Enter Alias ID" class="form-control" name="aliasId" formControlName="aliasId" />
-                            </div>
-                            <div class="form-group col-sm-6 col-xl-2 col-md-6 payment-col">
-                                <label>Accounts <span class="required">*</span></label>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.html:796]: Need to confirm

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html [hunk 9]
```diff
@@ -1,908 +1,938 @@
...
-                                <br>
-                                <sh-select #paymentFormAccountName [options]="bankAccounts$ | async" name="accountUniqueName" formControlName="accountUniqueName" [forceClearReactive]="forceClear$ | async" placeholder="Select Account" [ItemHeight]="33" autocomplete="off" [isRequired]="true">
-                                </sh-select>
-                            </div>
-                        </div>
-                        <ng-container formArrayName="userAmountRanges">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.html:802]: Need to confirm

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.html [hunk 10]
```diff
@@ -1,31 +1,90 @@
...
+                        <img hidden src="../../../assets/images/logo-icon.svg" class="company-icon mr-r1">
+                        <span  class="company-alice mr-r05 text-uppercase">{{companyInitials}}</span>
+                        <span class="d-flex justify-content-between w-100 company-full-name">{{selectedCompanyDetails?.name}}
+                            <span class="d-flex"> <i class="icon-down-new"></i> </span></span>
                     </h3>
                     <ul class="dropdown-menu user-dropdown-header" role="menu" *dropdownMenu>
                         <ng-container *ngTemplateOutlet="headDropdownMenu"></ng-container>
                     </ul>
                 </li>
+                <ul *ngIf="(currentCompanyBranches$ | async)?.length > 1" class="list-style-none  current-branch-name mr-l3  switch-branch-wrapper" >
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.html:21]: Need to confirm

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**